src/analysis/retail/priest/holy/CombatLogParser.tsx

```diff
@@ -85,6 +87,11 @@ class CombatLogParser extends CoreCombatLogParser {
     renew: Renew,
     prayerOfMending: PrayerOfMending,
 
+    //Items
```

Comment:
```suggestion
    // Items
```
make consistent with rest of file